src/main/java/com/doongji/nestalk/controller/v1/user/UserRestController.java

```diff
@@ -59,4 +56,11 @@
         String email = userService.findEmailByNameAndPhone(request.getName(), request.getPhone());
         return ResponseEntity.ok(new FindEmailResponse(email));
     }
+
+    @ApiOperation(value = "회원 정보 수정")
+    @PutMapping(path = "user/update")
```

Comment:
```suggestion
    @PutMapping(path = "user")
```
HTTP 메소드가 PUT이기 때문에, url에 update를 안써줘도 될 것 같아요.

Reply:
수정완료했습니다 착각했던부분 잘짚어주셨네요 감사합니다 :)